changelog.d/7232.doc

```diff
@@ -0,0 +1 @@
+Add systemd service, Let's Encrypt instructions and also clean up markdown, clarify the key usage and remove binary path.
```

Comment:
this is a bit misleading. synapse already has all these things.

why don't you just say

```suggestion
Update `turn-howto.md`. Contributed by @rubo77.
```